css/site.styl

```diff
@@ -322,6 +322,23 @@ body.menu-active
   cursor: pointer
   
 // ******************************************************
+
+// SOBRE NOSOTROS
+
+.prensa-files
```

Comment:
theres a utility class for inline lists. instead of putting `display: inline-block` on the `li` you add `class="u-inline-list` to the `<ul>`